persistence/schemas/de.cyface.persistence.DatabaseV6/1.json

```diff
@@ -0,0 +1,120 @@
+{
+  "formatVersion": 1,
+  "database": {
+    "version": 1,
+    "identityHash": "c3ec4780488bc825ddc6571e20a59607",
+    "entities": [
+      {
```

Comment:
I added the database schema generated by Room to the version control to ensure the schema is as expected.